source/includes/_teams.md

```diff
@@ -0,0 +1,216 @@
+# Teams
+
+## TeamMembership object definition
+
+> Request TeamMembership
+
+```shell
+{
+    "advisor": 1,
+    "is_primary": true|false
+}
+```
+
+| Parameter  | Type | Description                                                                 |
+|------------|------|-----------------------------------------------------------------------------|
+| advisor    | int  | Advisor id                                                                  |
+| is_primary | bool | Boolean value indicating if the advisor is the primary advisor for the team |
+
+> Response TeamMembership
+
+```shell
+{
+    "advisor": {
+        "id": 1,
+        "user": {
+            "id": 1,
+            "first_name": "First",
+            "last_name": "Last",
+            "email": "first_last@stratifi.com"
+        },
+        "company": 1,
+        "title": "",
+        "phone": "+11111111111",
+        "default_investor": 2,
+        "external_id": ""
+    },
+    "is_primary": true|false
+}
+```
+
+| Name       | Type   | Description                                                                 |
+|------------|--------|-----------------------------------------------------------------------------|
+| advisor    | object | [Advisor object](#advisor-object-definition)                                |
+| is_primary | bool   | Boolean value indicating if the advisor is the primary advisor for the team |
+
+## Team object definition
+
+> Team Object
+
+```shell
+{
+    "id": 1,
+    "name": "Some Team",
+    "external_id": "45912d",
+    "description": "Team description",
+    "company": 1,
+    "rep_codes": [],
+    "team_memberships": [
+        {...TeamMembership Object},
+        {...TeamMembership Object}
+    ]
+}
+```
+
+| Name             | Type        | Description                                                   |
+|------------------|-------------|---------------------------------------------------------------|
+| id               | int         | Team id                                                       |
+| name             | string      | Team name                                                     |
+| external_id      | string      | External id                                                   |
+| description      | string      | Team description                                              |
+| company          | int         | Company id                                                    |
+| rep_codes        | string list | List of rep codes                                             |
+| team_memberships | object list | List of [team memberships](#teammembership-object-definition) |
+
+## List Teams
+
+> List Teams
```

Comment:
can we add some filtering capabilities? e g by `external_id` like other endpoints.

See **Filtering Fields** in other files.

Reply:
yep, can check

Reply:
forgot to add those to docs